receiptui/src/androidTest/java/com/hyperwallet/android/ui/receipt/ListPrepaidCardReceiptsTest.java

```diff
@@ -481,4 +512,17 @@ private void setLocale(Locale locale) {
         }
         resources.updateConfiguration(configuration, resources.getDisplayMetrics());
     }
-}
+
+    private String dateConversion(String expectedDate, String timeAdjustment, String dateFormat,
+            int flag) {
+        SimpleDateFormat simpleDateFormat = new SimpleDateFormat(dateFormat + "HH:mm:ss");
+        simpleDateFormat.setTimeZone(TimeZone.getTimeZone("GMT"));
```

Comment:
Could you please confirm this hardcoded TIMEZONE with @skoong.

Reply:
@vinotb-altmtrk This is incorrect, you need to convert the time which is in the JSON file (GMT) to the timezone which is in the device for display.  There is already something that does this in ListUserReceiptsTest.java:278.  You just need to generalize this function to make it work for the Month header as well.

Reply:
You can leave the the input parameter as the date string, so it should look like this
dateConversion("2019-06-07T17:08:58", flags)

Reply:
done.